workshops/geoserver/intro/doc/source/install/suite.rst

```diff
@@ -0,0 +1,230 @@
+In this section we will install a virtual machine containing Boundless Suite.
+
+Prerequisites:
+
+* Please disable any programs on your system that use either 8080. (If this is not possible, please ask for alternative connection options.)
+* We strongly recommend you use a recent version of `Firefox <http://www.mozilla.org/en-US/firefox/new/>`_ as the browser on your host system. Using `Chrome <https://www.google.com/intl/en/chrome/browser/>`_ is acceptable, though an XML viewer extension such as `XV <https://chrome.google.com/webstore/detail/xv-%E2%80%94-xml-viewer/eeocglpgjdpaefaedpblffpeebgmgddk?hl=en>`_ will be required for some sections. Other browsers have not been tested and are not recommended.
+* Make sure you have administrative rights (Windows) or super-user privileges (Linux and OS X) on your system.
+
+.. _install.suite.vm:
+
+Install the virtual machine
+---------------------------
+
+Next we will install the Boundless Suite virtual machine. This virtual machine will run inside `VirtualBox`_ in the background of your system.
+
+.. note:: You must be able to run a 64-bit virtual machine. 32-bit machines are not supported.
+
+.. _VirtualBox: https://www.virtualbox.org/wiki/Downloads
+
+#. Download the Boundless Suite virtual machine from the `Boundless Connect <https://connect.boundlessgeo.com>`__ .
+   
+   * Boundless Connect is a free sign up to access Boundless Suite, and to access our training and workshop materials.
+   * The latest release of Boundless Suite is restricted to our customers, prior releases are available for free download.
+   * This download is included in your :file:`software` folder
+   
+   .. figure:: img/connect.png
+      :width: 100%
+      
+      Boundless Connect
+      
+#. Download and install the latest `VirtualBox`_. You may keep all defaults during the install.
+   
+   .. figure:: img/vbox.png
+
+      Installing VirtualBox for Windows
+
+   .. note:: During install on Windows, you may see a warning about your network interfaces. **This is expected.** Your network connections will be *temporarily* disconnected, and automatically reset after installation has completed. So be aware that you don't want to be utilizing your network connection (for example, downloading something) during the installation of VirtualBox.
+
+      .. figure:: img/vbox_network.png
+      
+         Network Interfaces warning
+
+#. Double-click the VM image file that you downloaded above (with the :file:`.ova` file extension). This will import the virtual machine into VirtualBox.
+
+   .. figure:: img/vbox_import.png
+      :width: 75%
+
+      Virtual machine details
+      
+#. Click :guilabel:`Import` to accept the defaults.
+
+#. You will now see the :guilabel:`BoundlessSuite` entry in the list of virtual machines in VirtualBox.
+
+   .. figure:: img/vbox_imported.png
+      :width: 100%
+      
+      A successful import of the training virtual machine
+         
+Determining hardware virtualization compatibility

```

Comment:
Let's just both be aware that if we update the training repo instructions, this won't come along for the ride, and we'll need to sync them again.